app/models/foreman_kubevirt/kubevirt.rb

```diff
@@ -150,6 +150,13 @@ def create_vm(args = {})
         volume.type = 'containerDisk'
         volumes << volume
       end
+      is_bootable = false
+      volumes_attributes.each do |_, v|
```

Comment:
you can simplified that to:
`raise ::Foreman::Exception.new(N_('It is not possible to set a bootable volume and image based provisioning.')) if args.key?("image_id") && volumes_attributes.any? { |_,v| v["bootable"] == "true" }`

Reply:
also, you can pull this out to a method and add a test to cover it.